workflow/controller/workflowpod.go

```diff
@@ -220,7 +225,7 @@ func (woc *wfOperationCtx) createWorkflowPod(nodeName string, mainCtr apiv1.Cont
 		woc.log.Infof("Failed to create pod %s (%s): %v", nodeName, nodeID, err)
 		return nil, errors.InternalWrapError(err)
 	}
-	woc.log.Infof("Created pod: %s (%s)", nodeName, created.Name)
+	woc.log.Infof("Created pod: %s (%s), serviceaccount=%s ", nodeName, created.Name, tmplServiceAccountName)
```

Comment:
Can we revert this line before merging? I feel serviceAccount is an arbitrary field that is included in this log message.

Reply:
This comment was unaddressed.